tests/Avalonia.Base.UnitTests/Media/GlyphTypefaceTests.cs

```diff
@@ -0,0 +1,447 @@
+
+            var typeface = new GlyphTypeface(new CustomPlatformTypeface(stream), FontSimulations.Bold);
+
+            Assert.Equal(FontWeight.Bold, typeface.Weight);
+            Assert.Equal(FontSimulations.Bold, typeface.FontSimulations);
+        }
+
+        [Fact]
+        public void Should_Apply_Oblique_Simulation()
+        {
+            var assetLoader = new StandardAssetLoader();
+
+            using var stream = assetLoader.Open(new Uri(s_InterFontUri));
+
+            var typeface = new GlyphTypeface(new CustomPlatformTypeface(stream), FontSimulations.Oblique);
+
+            Assert.Equal(FontStyle.Italic, typeface.Style);
+            Assert.Equal(FontSimulations.Oblique, typeface.FontSimulations);
+        }
+
+        [Fact]
+        public void Should_Apply_Combined_Simulations()
+        {
+            var assetLoader = new StandardAssetLoader();
+
+            using var stream = assetLoader.Open(new Uri(s_InterFontUri));
+
+            var typeface = new GlyphTypeface(new CustomPlatformTypeface(stream), 
+                FontSimulations.Bold | FontSimulations.Oblique);
+
+            Assert.Equal(FontWeight.Bold, typeface.Weight);
+            Assert.Equal(FontStyle.Italic, typeface.Style);
+            Assert.Equal(FontSimulations.Bold | FontSimulations.Oblique, typeface.FontSimulations);
+        }
+
+        [Fact]
+        public void Should_Have_TypographicFamilyName()
+        {
+            var assetLoader = new StandardAssetLoader();
+
+            using var stream = assetLoader.Open(new Uri(s_InterFontUri));
+
+            var typeface = new GlyphTypeface(new CustomPlatformTypeface(stream));
+
+            Assert.NotNull(typeface.TypographicFamilyName);
+        }
+
+        [Fact]
+        public void Should_Have_FamilyNames_Dictionary()
+        {
+            var assetLoader = new StandardAssetLoader();
+
+            using var stream = assetLoader.Open(new Uri(s_InterFontUri));
+
+            var typeface = new GlyphTypeface(new CustomPlatformTypeface(stream));
+
+            Assert.NotNull(typeface.FamilyNames);
+            Assert.NotEmpty(typeface.FamilyNames);
+        }
+
+        [Fact]
+        public void Should_Have_FaceNames_Dictionary()
+        {
+            var assetLoader = new StandardAssetLoader();
+
+            using var stream = assetLoader.Open(new Uri(s_InterFontUri));
+
+            var typeface = new GlyphTypeface(new CustomPlatformTypeface(stream));
+
+            Assert.NotNull(typeface.FaceNames);
+            Assert.NotEmpty(typeface.FaceNames);
+        }
+
+        [Fact]
+        public void Should_Have_SupportedFeatures()
+        {
+            var assetLoader = new StandardAssetLoader();
+
+            using var stream = assetLoader.Open(new Uri(s_InterFontUri));
+
+            var typeface = new GlyphTypeface(new CustomPlatformTypeface(stream));
+
+            var features = typeface.SupportedFeatures;
+
+            Assert.NotNull(features);
```

Comment:
This should be `NotEmpty`, the property will never be null even if there are no features.